include/roboteam_ai/control/PosController.h

```diff
@@ -32,14 +33,11 @@ class PosController {
         bool customCanMoveInDefenseArea = true;
 
         // PID functions
-        PID xpid = PID(0.0, 0.0, 0.0);
-        PID ypid = PID(0.0, 0.0, 0.0);
+        PidTwoAxesController pid = PidTwoAxesController();
```

Comment:
What is the relation between the PosController and PidTwoAxesController? It is unclear for me, so this variable needs a comment which explains this relation.